src/cmd/go/internal/search/search.go

```diff
@@ -52,6 +52,11 @@ func (m *Match) IsLiteral() bool {
 	return !strings.Contains(m.pattern, "...") && !m.IsMeta()
 }
 
+// ContainsWildcard reports whether the pattern contains any wildcard.
+func (m *Match) ContainsWildcard() bool {
```

Comment:
I see a bunch of places in the cmd/go codebase that use `strings.Contains(x, "...")` ([here's an example](https://github.com/golang/go/blob/fe26dfadc3630617d133b8c94bcb2ccb2e85dc1b/src/cmd/go/internal/modload/load.go#L259)) so I think you should probably write that rather than adding this new method.